src/main/java/net/twasi/obsremotejava/OBSCommunicator.java

```diff
@@ -1,6 +1,7 @@
 package net.twasi.obsremotejava;
 
 import com.google.gson.*;
+import java.lang.reflect.Modifier;
```

Comment:
Unused Import?

Reply:
Perfect timing XD
Let's create a new PR for this, since this was just merged.